pkg/client.go

```diff
@@ -45,6 +45,7 @@ func (client *ClickHouseClient) Query(query string) (*Response, error) {
 
 	if client.settings.Instance.BasicAuthEnabled {
 		password, _ := client.settings.Instance.DecryptedSecureJSONData["basicAuthPassword"]
+		req.SetBasicAuth(client.settings.Instance.BasicAuthUser, password)
```

Comment:
@Slach in the scenario mentioned regarding a proxy, shouldn't `X-ClickHouse-User`  and password be set from the header parameter? That way the basic auth could be a different value than the click house user and password?

Reply:
i think it should resolve during implements https://github.com/Vertamedia/clickhouse-grafana/issues/348